setup.py

```diff
@@ -22,6 +22,7 @@
         "pytz>=2016.10",
         "shortuuid>=0.4.3",
         "tabulate>=0.7.7",
+        "pathlib2; python_version < '3.4'",
```

Comment:
This requires the newer version of setuptools. I think it should be fine for most of the users. I'll test this for python2.7 and python3.5 after pushing to pypi.